polymorphic_test.ts

```diff
@@ -28,6 +28,7 @@ test({
 
       fail("polymorphicClassFromJSON did not error with no context");
     } catch (e) {
+      assert(e instanceof Error);
```

Comment:
oh if this works I should change what I did, this is cleaner IMHO

Reply:
This works!